process/database/elasticSearchConnector.go

```diff
@@ -151,6 +150,32 @@ func (esc *elasticSearchConnector) doSearchRequest(query object, index string, s
 	return decodedBody, nil
 }
 
+func (esc *elasticSearchConnector) doSearchRequestTx(address string, index string, size int) (object, error) {
+	res, err := esc.client.Search(
+		esc.client.Search.WithIndex(index),
+		esc.client.Search.WithSize(size),
+		esc.client.Search.WithQuery("sender%"+address+"+receiver%"+address),
```

Comment:
I think it would be better if you would use `fmt.Sprintf("sender:%s OR receiver:%s", address, address)` or something similar. I know it works the way it is now, but for a better code readability.

Reply:
If I have `fmt.Sprintf`  I will have this `fmt.Sprintf("sender%%%s+receiver%%%s", address, address)` whitch is not so easy to understand. So I will let as it is for more readability.

Reply:
done.